apps/examples/testcase/ta_tc/task_manager/itc/itc_taskmanager_main.c

```diff
@@ -146,99 +201,146 @@ void itc_taskmanager_check_dedicate_permission_p(void)
 	TC_SUCCESS_RESULT();
 }
 
-void itc_taskmanager_user1_main(void)
+void tm_builtin_main(void)
```

Comment:
Why is this name builtin?

Reply:
Function name should represent what it works. I think this is not.

Reply:
changed it